bddcli/authoring/given.py

```diff
@@ -24,7 +25,13 @@ def invoke(self, call):
 
     def kill(self, s=signal.SIGTERM):
         call = self.current_call
-        os.killpg(os.getpgid(call.process.pid), s)
+        if os.name == "nt":
```

Comment:
Please create a new module named `platform_.py` and move all platform-specific stuff into the `platform_.py`.

platform_.py
```python  

def killpg(...):
    ....

```

given.py
```python
from .platform_ import killpg

...

    def kill(self, s=signal.SIGTERM):
        call = self.current_call
        killpg(call.process.pid, s)

```